src/shared/components/Resources/Links.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { Resource, PaginationSettings } from '@bbp/nexus-sdk';
+import { Spin, Card, Icon } from 'antd';
+
+import AnimatedList from '../Animations/AnimatedList';
+import { ResourceLink } from '@bbp/nexus-sdk/lib/Resource/types';
+import ResourceListItem from './ResourceItem';
+import { labelOf } from '../../utils';
+import { LinksState } from '../../store/reducers/links';
+
+import './links-container.less';
+import { LinkDirection } from '../../store/actions/nexus/links';
```

Comment:
Shouldn't this type be coming from the SDK rather than a redux action?

Reply:
I didn't think about that because the SDK has two separate methods getIncoming and getOutoing, but maybe its better to have one getLinks("incoming" : "outgoing) and then this enum would have a place there. What do you think?